public/index.html

```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon_16x16.ico" sizes="16x16">
-    <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon_32x32.ico" sizes="32x32">
-    <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon_48x48.ico" sizes="48x48">
+    <link rel="icon" type="image/png" href="%PUBLIC_URL%/assets/images/favicon_16x16.ico" sizes="16x16">
+    <link rel="icon" type="image/png" href="%PUBLIC_URL%/assets/images/favicon_32x32.ico" sizes="32x32">
+    <link rel="icon" type="image/png" href="%PUBLIC_URL%/assets/images/favicon_48x48.ico" sizes="48x48">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Small business hero - hier kannst Du Deinen Laden unterstützen"
```

Comment:
Let's align this with the title defined in manifest.json
```suggestion
      content="Unterstütze Deinen Laden um die Ecke"
```